### ISSUE TITLE
Highlight delay

### ISSUE THREAD

**Issue by rmcarder** [EAPTSD/EAP-Map#9]
When moving the mouse quickly, the highlight lags a bit in catching up. We had this problem on CCUSA, but Lukas fixed it. I'll look to see if I can bring that solution over.

**Comment by ostermanj** [comment]
Yeah, thanks. I noticed that. Probably a setTimeout, clearTimeout solution to delay its taking effect by a moment.
